src/requirements.cpp

```diff
@@ -243,6 +253,54 @@ int requirement_data::print_list( WINDOW *w, int ypos, int xpos, int width, nc_c
     return ypos - oldy;
 }
 
+template<typename T>
```

Comment:
Why is this 2x as big as print_list?  Ideally we wouldn't have two methods for this at all, but one that can handle both use cases.